COPYRIGHT.txt

```diff
@@ -303,6 +303,12 @@ Comment: Tangent Space Normal Maps implementation
 Copyright: 2011, Morten S. Mikkelsen
 License: Zlib
 
+Files: ./thirdparty/misc/ok_color.h
+ ./thirdparty/misc/ok_color_shader.h
+Comment: OK Lab color space
```

Comment:
Upstream seems to spell it "Oklab": https://bottosson.github.io/posts/oklab/

Reply:
Will close when revised.